### PR TITLE
[Docs] `extensions`: improved cases for using `@/...`

### PR DESCRIPTION
**Pull request by wenfangdu**
Add cases to show more accurate behavior of `always`.

### CODE REVIEW

File: docs/rules/extensions.md [hunk 1]
```diff
@@ -120,7 +122,9 @@ import bar from './bar.json';
 
 import Component from './Component.jsx';
 
-import * as path from 'path';
```
Comment (ljharb) [inline docs/rules/extensions.md:123]: why is this one removed?

File: docs/rules/extensions.md [hunk 2]
```diff
@@ -109,6 +109,8 @@ import foo from './foo';
 import bar from './bar';
 
 import Component from './Component';
+
+import foo from '@/foo'
```
Comment (ljharb) [inline docs/rules/extensions.md:113]: this isn't correct - the tests currently ensure that with "always", this should indeed omit the extension

**Review by ljharb — CHANGES_REQUESTED** [review]

File: docs/rules/extensions.md (hunk 2 above)
Comment (wenfangdu) [reply docs/rules/extensions.md]: @ljharb [This test](https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L507-L523) will throw.

Comment (wenfangdu) [reply docs/rules/extensions.md]: @ljharb [This test](https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L507-L523) will throw.

File: docs/rules/extensions.md (hunk 1 above)
Comment (wenfangdu) [reply docs/rules/extensions.md:123]: @ljharb I've restored it, please have a look.

File: docs/rules/extensions.md (hunk 2 above)
Comment (ljharb) [reply docs/rules/extensions.md]: and this one won't https://github.com/import-js/eslint-plugin-import/blob/b236748e39d97dae4be873e06b77c411e178b654/tests/src/rules/extensions.js#L342 - maybe due to `ignorePackages` though? If so, let's be sure to include this example under that section as well.

Comment (wenfangdu) [reply docs/rules/extensions.md]: @ljharb Seems like it has been added:

https://github.com/import-js/eslint-plugin-import/blob/b743a650e9d474fcd5a4252b01c522adcc11b65c/tests/src/rules/extensions.js#L352-L356

Comment (wenfangdu) [reply docs/rules/extensions.md]: @ljharb What else do I need to do?

Comment (ljharb) [reply docs/rules/extensions.md]: I meant to add something in the rule docs :-)

Comment (wenfangdu) [reply docs/rules/extensions.md]: I've added two more cases:

- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R152
- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R164

Comment (wenfangdu) [reply docs/rules/extensions.md]: @ljharb I've added two more cases:

- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R152
- https://github.com/import-js/eslint-plugin-import/pull/2140/commits/99f31587b18854e05959a1ede2cca1cc7e2772ce#diff-e936099f904c0062296b824ed9e27946c5da628c70d5476d55a132869ea67183R164

**Review by ljharb — APPROVED** [review]

**Pull request merged**